_board/spresense.md

```diff
@@ -13,14 +13,38 @@ features:
   - Arduino Shield Compatible
 ---
 
-The Spresense project consists of a Arduino compatible board with Sony’s high performance CXD5602 micro-controller. The CXD5602 has built-in GPS and high-resolution audio capabilities.
+The Spresense project consists of a Arduino compatible board with Sony's high performance CXD5602 micro-controller. The CXD5602 has built-in GPS and high-resolution audio capabilities.
 
 The main board uses a processor developed by Sony for IoT and sensing applications. The main board can be operated alone or with the extension board.
 
-The Spresense uses Sony’s new chipset on the main board: * The CXD5602 System on Chip (SoC) multi core processor with GNSS * The CXD5247 power management and audio analog interface chip
+The Spresense uses Sony's new chipset on the main board:
+
+* The CXD5602 System on Chip (SoC) multi core processor with GNSS
+* The CXD5247 power management and audio analog interface chip
 
 The Spresense extension board is a board which extends the interfaces compared to the Spresense main board. The Spresense main board and the Spresense extension board are connected by a Board-to-Board (B-2-B) connector. The Spresense extension board has Arduino Uno pin compatible shape and pin socket locations. However, there are some differences compared to the Arduino Uno. 
 
+## Technical details
+
+* CXD5602 Processor (Arm Cortex-M4F 156 MHz)
+* 8 MB Flash memory, SRAM 1.5 MB
+* PCB with small footprint
+* Dedicated camera connector
+* GNSS (GPS) antenna
+* Pins and LEDs
+  * Multiple GPIO (UART, SPI, I2C, I2S)
+  * 2 ADC channels
```

Comment:
There are 2 ADC channels on the main board, but 6 channels is on the extension board. Maybe it is worth adding this information here.

Reply:
Thanks